charts/llm-d-modelservice/templates/decode-lws.yaml

```
@@ -6,17 +6,23 @@ metadata:
   labels:
     {{- include "llm-d-modelservice.labels" . | nindent 4 }}
     {{- include "llm-d-modelservice.decodelabels" . | nindent 4 }}
+  annotations:
+    leaderworkerset.sigs.k8s.io/subgroup-exclusive-topology: kubernetes.io/hostname
 spec:
-  {{- if not .Values.decode.autoscaling.enabled }}
+  {{- if not (and .Values.decode.autoscaling .Values.decode.autoscaling.enabled) }}
```

I think we can only use the `.Values.decode.autoscaling.enabled` here:
```suggestion
  {{- if not Values.decode.autoscaling.enabled }}
```
I think this because you can have an autoscaling values config without wanting to use it